test/e2e/e2e.go

```diff
@@ -317,17 +317,17 @@ func setupSuite() {
 
 // logClusterImageSources writes out cluster image sources.
 func logClusterImageSources() {
-	masterImg, nodeImg, err := lookupClusterImageSources()
+	controlPlaneNodeImg, dataPlaneNodeImg, err := lookupClusterImageSources()
```

Comment:
I don't know what data plane nodes mean in k8s cluster.
worker nodes as https://kubernetes.io/docs/concepts/overview/components/ ?

Reply:
It was a suggestion I made @oomichi 

https://kubernetes.io/docs/reference/glossary/?all=true#term-data-plane

Maybe there is another word, or `workerNodeImg` is good instead?

Reply:
Looking at it now worker does seem better to correlate with worker nodes. I am open to other suggestions.

Reply:
@oomichi @MorrisLaw do you feel all instances of data-plane would be better as worker/workerNode?

Reply:
I think if we go with `worker` instead of `data-plane`. Then yeah it should be `workerNodeImg`, `worker-node` etc.  @mgutierrez98

Reply:
Will get this change implemented.